website/src/app/user-detail/user-detail.component.html

```diff
@@ -0,0 +1,36 @@
+<app-navigation></app-navigation>
+
+
+<div class="user-detail-container" fxLayout="column" fxLayoutAlign="space-between">
+    <form [formGroup]="updateForm">
```

Comment:
This largely is a copy of create-user form, would be nice to see an attempt to reduce duplication. This way a change to User (i.e. adding a field) makes you change in two places not one.